### ISSUE TITLE
Install Error - Failed to install modules

### ISSUE THREAD

**Issue by YD1722** [adopted-ember-addons/ember-electron#392]
OS : Windows 7
Node version : 6.9.2
Ember-cli :3.3

```
ember install ember-electron

npm: Installed ember-electron
installing ember-electron
  create ember-electron\.compilerc
  create ember-electron\main.js
  create ember-electron\test-main.js
  create testem-electron.js
Project needs forge config
Failed to install modules: ["electron-prebuilt-compile@5.0.1"]

With output: Exited with status 1

```

**Comment by jacobq** [comment]
Hmm, there is no "electron-prebuilt-compile@5.0.1" published on NPM (see lament https://github.com/electron-userland/electron-prebuilt-compile/issues/77)

I just ran `ember install ember-electron` on an empty/new ember project on Windows, and it works fine. Could you please share your package.json file? (Or at least the deps/devDeps entries for packages with `electron` in their names)
```
ember-cli: 3.9.0
node: 12.1.0
os: win32 x64
```

**Comment by YD1722** [comment]
`{
  "name": "",
  "version": "0.0.0",
  "private": true,
  "description": "",
  "license": "",
  "author": "",
  "directories": {
    "doc": "doc",
    "test": "tests"
  },
  "repository": "",
  "scripts": {
    "build": "ember build",
    "start": "ember serve",
    "test": "ember test"
  },
  "devDependencies": {
    "broccoli-asset-rev": "^2.7.0",
    "broccoli-concat": "^3.2.2",
    "broccoli-uglify-js": "^0.2.0",
    "datasaur-sorter": "^1.0.0",
    "ember-ajax": "^3.0.0",
    "ember-browserify": "^1.2.0",
    "ember-cli": "3.3.0",
    "ember-cli-app-version": "^3.0.0",
    "ember-cli-babel": "^6.6.0",
    "ember-cli-bootstrap-datepicker": "^0.6.1",
    "ember-cli-dependency-checker": "^2.0.0",
    "ember-cli-eslint": "^4.2.1",
    "ember-cli-htmlbars": "^2.0.1",
    "ember-cli-htmlbars-inline-precompile": "^1.0.0",
    "ember-cli-inject-live-reload": "^1.4.1",
    "ember-cli-qunit": "^4.3.2",
    "ember-cli-sass": "5.2.0",
    "ember-cli-shims": "^1.2.0",
    "ember-cli-sri": "^2.1.0",
    "ember-cli-uglify": "^2.0.0",
    "ember-export-application-global": "^2.0.0",
    "ember-load-initializers": "^1.1.0",
    "ember-maybe-import-regenerator": "^0.1.6",
    "ember-resolver": "^4.0.0",
    "ember-source": "3.3.0",
    "eslint-plugin-ember": "^5.0.0",
    "filepath": "^1.1.0",
    "fin-hypergrid": "file:./lib/hypergrid/core-3.0.3",
    "fin-hypergrid-grouped-header-plugin": "1.2.4",
    "datasaur-filter": "file:./lib/hypergrid/datasaur-filter-sort-custom-3.0.0",
    "datasaur-indexed": "file:./lib/hypergrid/datasaur-indexed-custom",
    "loader.js": "^4.2.3",
    "qunit-dom": "^0.6.2"
  },
  "engines": {
    "node": "6.* || 8.* || >= 10.*"
  }
}
`

**Comment by YD1722** [comment]
```
{
  "name": "test",
  "version": "0.0.0",
  "private": true,
  "description": "test",
  "license": "",
  "author": "",
  "directories": {
    "doc": "doc",
    "test": "tests"
  },
  "repository": "",
  "scripts": {
    "build": "ember build",
    "start": "ember serve",
    "test": "ember test"
  },
  "devDependencies": {
    "broccoli-asset-rev": "^2.7.0",
    "broccoli-concat": "^3.2.2",
    "broccoli-uglify-js": "^0.2.0",
    "datasaur-sorter": "^1.0.0",
    "ember-ajax": "^3.0.0",
    "ember-browserify": "^1.2.0",
    "ember-cli": "3.3.0",
    "ember-cli-app-version": "^3.0.0",
    "ember-cli-babel": "^6.6.0",
    "ember-cli-bootstrap-datepicker": "^0.6.1",
    "ember-cli-dependency-checker": "^2.0.0",
    "ember-cli-eslint": "^4.2.1",
    "ember-cli-htmlbars": "^2.0.1",
    "ember-cli-htmlbars-inline-precompile": "^1.0.0",
    "ember-cli-inject-live-reload": "^1.4.1",
    "ember-cli-qunit": "^4.3.2",
    "ember-cli-sass": "5.2.0",
    "ember-cli-shims": "^1.2.0",
    "ember-cli-sri": "^2.1.0",
    "ember-cli-uglify": "^2.0.0",
    "ember-export-application-global": "^2.0.0",
    "ember-load-initializers": "^1.1.0",
    "ember-maybe-import-regenerator": "^0.1.6",
    "ember-resolver": "^4.0.0",
    "ember-source": "3.3.0",
    "eslint-plugin-ember": "^5.0.0",
    "filepath": "^1.1.0",
    "fin-hypergrid": "file:./lib/hypergrid/core-3.0.3",
    "fin-hypergrid-grouped-header-plugin": "1.2.4",
    "datasaur-filter": "file:./lib/hypergrid/datasaur-filter-sort-custom-3.0.0",
    "datasaur-indexed": "file:./lib/hypergrid/datasaur-indexed-custom",
    "loader.js": "^4.2.3",
    "qunit-dom": "^0.6.2"
  },
  "engines": {
    "node": "6.* || 8.* || >= `10.*"`
  }
}

```

**Comment by jacobq** [comment]
I tried to reproduce this again using node 6.x and ember-cli 3.3 but so far have not been able to. I think it is happening because electron-forge is attempting to use the version number associated with the latest electron release (5.0.1) with the electron-prebuilt-compile package, which is behind. 
(See https://github.com/electron-userland/electron-forge/blob/5.x/src/api/import.js#L178-L180)

Could you enable debugging, try again, and post the log here? 
`DEBUG=electron-forge:*,ember-electron:* ember install ember-electron`

**Comment by PavloNaumenko** [comment]
@jacobq I have the same issue. It looks like changes from the master branch haven't been published to the npm (with a new tag). 

@YD1722 You can make changes in package.json till the project maintainers don't publish it :
`    "ember-electron": "https://github.com/adopted-ember-addons/ember-electron.git",`

**Comment by jacobq** [comment]
> It looks like changes from the master branch haven't been published to the npm (with a new tag).

Yes, and sorry for inconvenience. We're in the process of changing maintainers. However, I'm still puzzled as to why the problem is happening / what [changes in master](https://github.com/adopted-ember-addons/ember-electron/compare/v2.10.0...master) fix it. @PavloNaumenko could you share some debug output? Is it because of older electron-forge?  If so, why can't I reproduce it? Lockfile issue?

**Comment by PavloNaumenko** [comment]
@jacobq Hm, I can't reproduce it again. Maybe it was an npm cache?
